src/App/Resources/NamedStyles/AppWindow.xaml

```diff
@@ -124,6 +126,9 @@
                         <Trigger Property="ResizeMode" Value="NoResize">
                             <Setter TargetName="PART_MAXIMIZE_RESTORE" Property="Visibility" Value="Collapsed" />
                         </Trigger>
+                        <Trigger Property="WindowState" Value="Maximized">
+                            <Setter TargetName="MainContentPresenter" Property="Margin" Value="0,0,15,40" />
+                        </Trigger>
```

Comment:
Does it work in different screen resolutions?

Reply:
Did a minor improvement and it works with all screen resolutions.